### PR TITLE
Implement Test1 using Page Object #19

### LINKED ISSUE #19: Implement Test1 using Page Object
Test1:
Пользователь может создать  новый  комментарий при заполнении всех обязательных полей

https://docs.google.com/spreadsheets/d/1A3_5XFjxRlkfuSTgRIa81SMhW56siWPfEfo-_7YgLPQ/edit#gid=0

### PR DESCRIPTION
**Pull request by akdcstar**
Test1:
Пользователь может создать новый комментарий при заполнении всех обязательных полей

Closes #19

### CODE REVIEW

File: src/main/java/net/comments/ft/Test1.java [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+        commentsPage.commentsFrom(4);
+
+        MatcherAssert.assertThat("New comment is present", commentsPage.currentComments().hasCommentWithId(888));
+    }
+
+    @BeforeMethod
+    public void createDriver() {
+        this.driver.define();
+    }
+//    @AfterMethod
```
Comment (extsoft) [inline src/main/java/net/comments/ft/Test1.java]: Please uncomment or delete the commented code.

File: src/main/java/net/comments/objects/BCommentsPage.java [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+    public CommentsTable currentComments() {
+        return new BCommentsTable(driver);
+    }
+
+    public void commentsFrom(int page) {
+        this.driver.findElement(By.linkText(String.format("%s", page))).click();
+
+    }
+//    public CommentsTable commentsFrom(int page) {
```
Comment (extsoft) [inline src/main/java/net/comments/objects/BCommentsPage.java]: Please delete commented code.

File: src/main/java/net/comments/objects/NewCommentBody.java [hunk 3]
```diff
@@ -0,0 +1,11 @@
+package net.comments.objects;
+
+public interface NewCommentBody {
+    void fillCommentText();
```
Comment (extsoft) [inline src/main/java/net/comments/objects/NewCommentBody.java]: Please pass a `text` as an argument for this method.

File: src/main/java/net/comments/selenium/CommentsDriver.java [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+public class CommentsDriver implements WebDriver {
+    private final WebDriver[] driver = new WebDriver[1];
+
+    private WebDriver driver() {
+        return this.driver[0];
+    }
+
+    public void define(WebDriver webDriver) {
+        this.driver[0] = webDriver;
+
```
Comment (extsoft) [inline src/main/java/net/comments/selenium/CommentsDriver.java]: Remove empty line.

File: src/main/java/net/comments/objects/NewCommentBody.java [hunk 5]
```diff
@@ -0,0 +1,11 @@
+package net.comments.objects;
+
+public interface NewCommentBody {
```
Comment (extsoft) [inline src/main/java/net/comments/objects/NewCommentBody.java]: Please rename this to `ModifyCommentPage` as this page is the same for editing and creation.

Comment (extsoft) [inline src/main/java/net/comments/objects/NewCommentBody.java]: Add one more method here: `NewCommentActions modifyActions()`

File: src/main/java/net/comments/objects/NewCommentActions.java [hunk 6]
```diff
@@ -0,0 +1,9 @@
+package net.comments.objects;
+
+public interface NewCommentActions {
```
Comment (extsoft) [inline src/main/java/net/comments/objects/NewCommentActions.java]: I suggest to rename it to `ModifyCommentActions()`.

**Review by extsoft — CHANGES_REQUESTED** [review]
@akdcstar Good work. Please take a look at a couple of comments.

**Comment by akdcstar** [comment]
Fixed all comments. 
@extsoft please, review.

**Review by extsoft — APPROVED** [review]
@akdcstar you still like empty lines which aren't needed.  However, excellent job!

**Comment by akdcstar** [comment]
@extsoft I'll check empty lines and will fix this in next commit.

**Comment by extsoft** [comment]
@akdcstar please remember about them next time you will commit something.

**Pull request merged**